packages/secure_storage/amplify_secure_storage/android/build.gradle

```diff
@@ -54,7 +54,7 @@ dependencies {
     implementation 'androidx.security:security-crypto:1.0.0'
     // TODO(Jordan-Nelson): remove once security-crypto:1.1.0 is stable.
     // See https://github.com/aws-amplify/amplify-flutter/issues/2640
-    implementation 'com.google.crypto.tink:tink-android:[1.8.0'
+    implementation 'com.google.crypto.tink:tink-android:[1.8.0, )'
```

Comment:
shall we use an upper bound like [1.8.0, 2.0.0)?

Reply:
We don't have any need for an upper bound. We do not use this dependency directly. It is a transitive dependency via `androidx.security:security-crypto`. `androidx.security:security-crypto` depends on an older version (~1.6.0) which logs a confusing message every time a key set is created. We need a version greater than or equal to 1.8 which is when that log was removed but we do not need to have nay constraint on the upper bound.